osu.Game/Overlays/OverlayRulesetTabItem.cs

```diff
@@ -35,7 +35,9 @@ protected virtual Color4 AccentColour
         [Resolved]
         private OverlayColourProvider colourProvider { get; set; }
 
-        private readonly OsuSpriteText text;
+        private readonly Drawable icon;
+
+        public LocalisableString TooltipText => Value.Name;
```

Comment:
This does not match what web does. On web, only the icon itself shows the ruleset name in the tooltip when hovered. This implementation also makes it show when hovering over the difficulty count.

See this screenshot:

![image](https://user-images.githubusercontent.com/20418176/166196522-5f6e1cb6-1293-4571-809d-ec26e7fa3dce.png)

Web doesn't show the tooltip when you point the cursor where I do on the screenshot above.

Reply:
I find the web implementation quite weird here. Generally I'd expect the tooltip to either cover the whole element, or only the icon portion to be clickable (not the number) as everything is floating in the first place.

Reply:
Hmm, I can agree with not showing the tooltip when hovering over the counter, and also not make the tab clickable from the number itself. But the latter may be a complicated one to solve given that the counter is part of the tab item, which is designed to be clickable. Unsure.

Reply:
Either way, tooltips are generally supposed to give you context about what you're clicking so I don't think it's too bad having it show for the full area.

(that said, we go against this with the weird implementation of tooltips on the song select leaderboard, but that's a bit of a temporary stand-in for proper score card popovers..)

Reply:
I'm fine with moving forward with the current version. I was mostly pointing it out as it seemed relevant in a "design parity with stable" PR.